state/initialize.go

```diff
@@ -219,14 +240,10 @@ func Initialize(args InitializeParams) (_ *State, err error) {
 	ops = append(ops, modelOps...)
 
 	if err := st.runTransaction(ops); err != nil {
-		return nil, errors.Trace(err)
-	}
-	controllerTag := names.NewControllerTag(args.ControllerConfig.ControllerUUID())
-	if err := st.start(controllerTag); err != nil {
```

Comment:
IIRC, NewState is calling st.start(), and this was then duplicating that call and triggering start to be called twice.